lib/final-api/helpers/ddtf_helpers.rb

```diff
@@ -17,14 +18,14 @@ def build_config(test_data, tsd)
               },
               name: tsd['name'],
               description: test_data.get_ikey('Description'),
-              branch: test_data.get_ikey('Package'),
+              branch: branch.get_ikey('Git') || branch.get_ikey('Unc'),
```

Comment:
maybe I am missing something...
The code is not readable and there is not one source of true. Could you define branch method in tsd_utils and use it here.
...maybe the same for `strategy`

Isn't the right solution to bupt build_config to tsd_utils?